src/components/DateRangePicker.js

```diff
@@ -256,35 +336,38 @@ const DatePicker = React.createClass({
             type={this.state.showCalendar ? 'caret-up' : 'caret-down'}
           />
         </div>
-        <div style={styles.calendarWrapper}>
-          <div style={styles.calendarHeader}>
-            <Icon
-              onClick={this._handlePreviousClick}
-              size={20}
-              style={styles.calendayHeaderNav}
-              type='caret-left'
-            />
-            <div>
-              {moment(this.state.currentDate, 'X').format('MMMM YYYY')}
+        <div style={styles.optionsWrapper}>
+          {this.props.showDefaultRanges && this._renderDefaultRanges()}
```

Comment:
This reads a little weird to me but I _think_ I like it better than throwing in the nulls in the ternary...